docs/versioned_docs/version-v1.5.0/features/hosting/formatting.md

```diff
@@ -17,8 +17,20 @@ PM > Install-Package Arcus.WebApi.Hosting
 ## Restricting JSON format
 We have provided an extension that will allow you to restrict your input and output formatting to only JSON formatting (Only the `SystemTextJsonInputFormatter` will remain). This means that all other incoming content will result in `UnsupportedMediaType` failures and outgoing content will fail to serialize back to the sender. With this functionality, you'll be sure that you only have to deal with JSON.
 
-Following example shows you where you can configure this:
+Following example shows you where you can configure this in your `Program.cs`:
 
+```csharp
+using Microsoft.AspNetCore.Mvc;
+
+builder.Services.AddControllers(options =>
+{
+    options.OnlyAllowJsonFormatting();
+});
+```
+
+<details>
```

Comment:
These changes needs to be added to the `/preview` docs. Since we already released this, and it can be used here, we could change it in the v1.5, indeed, but it should be foremost be changed in the preview docs first.
See the [contribution guide](https://github.com/arcus-azure/.github/blob/master/CONTRIBUTING.md#documentation) for more information on this.